src/plugins/walletConnectToDapps/components/modals/connect/ConnectContent.tsx

```diff
@@ -56,14 +58,25 @@ export const ConnectContent: React.FC<ConnectContentProps> = ({
     defaultValues: { uri: initialUri },
   })
 
+  const blurInput = useCallback(() => {
+    inputRef.current?.blur()
+  }, [])
+
   const handleQrScanSuccess = useCallback(
     (uri: string) => {
       setValue('uri', uri)
       toggleQrCodeView()
+      // Blur the input to close keyboard on mobile
+      setTimeout(blurInput, 100)
```

Comment:
We probably don't need a `setTimeout` here as we are setting `uri` manually, you couldn't test it as camera was buggy on iOS missing the permission, but you can test it in android deploying to `neo` or `gome` or any other env having https activated